src/main/java/ca/team2706/frc/robot/Robot.java

```diff
@@ -127,8 +130,10 @@ private void selectorInit() {
 
         // Check to see if the command exists in the desired index
         if (DriveBase.getInstance().getAnalogSelectorIndex() < commands.length && commands[index] != null) {
+            currentCommand = commands[DriveBase.getInstance().getAnalogSelectorIndex()];
             commands[DriveBase.getInstance().getAnalogSelectorIndex()].start();
```

Comment:
It might be better to save the current command, then start it at the end of the if statement, rather than duplicating the same selection logic.

```Java
if(condition) {
    currentCommand = <selection>;
}
else if(second condition) {
    currentCommand = <fallback command>
}

if(currentCommand != null) {
    currentCommand.start();
}
```